LHS C test incompatible pointer types

I'm seeing the following compiler warning for the LHS C test. Looks like something was changed but the test was not updated? I'm surprised the test is actually passing.

```C
../src/lhs/C/lhst.c: In function ‘main’:
../src/lhs/C/lhst.c:30:39: warning: passing argument 4 of ‘lhs_ihs’ from incompatible pointer type [-Wincompatible-pointer-types]
   30 |     lhs_ihs(n_dimen, n_points, &seed, (ipc_**)X, &control, &inform, &data);
      |                                       ^~~~~~~~~
      |                                       |
      |                                       ipc_ ** {aka int **}
In file included from ../src/lhs/C/lhst.c:8:
../include/galahad_lhs.h:173:21: note: expected ‘ipc_ *’ {aka ‘int *’} but argument is of type ‘ipc_ **’ {aka ‘int **’}
  173 |               ipc_ *X,
      |               ~~~~~~^
[679/1630] Compiling C object lhst_c_single.p/src_lhs_C_lhst.c.o
../src/lhs/C/lhst.c: In function ‘main’:
../src/lhs/C/lhst.c:30:39: warning: passing argument 4 of ‘lhs_ihs’ from incompatible pointer type [-Wincompatible-pointer-types]
   30 |     lhs_ihs(n_dimen, n_points, &seed, (ipc_**)X, &control, &inform, &data);
      |                                       ^~~~~~~~~
      |                                       |
      |                                       ipc_ ** {aka int **}
In file included from ../src/lhs/C/lhst.c:8:
../include/galahad_lhs.h:173:21: note: expected ‘ipc_ *’ {aka ‘int *’} but argument is of type ‘ipc_ **’ {aka ‘int **’}
  173 |               ipc_ *X,
      |               ~~~~~~^
```

Compiler doesn't recognise (ipc_)*  

hscs.c: In function ‘main’:
lhscs.c:30:39: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
   30 |     lhs_ihs(n_dimen, n_points, &seed, (ipc_)* X, &control, &inform, &data);
      |                                       ^
lhscs.c:30:39: warning: passing argument 4 of ‘lhs_ihs’ makes pointer from integer without a cast [-Wint-conversion]
   30 |     lhs_ihs(n_dimen, n_points, &seed, (ipc_)* X, &control, &inform, &data);

--- a/GALAHAD.jl/test/test_lhs.jl
+++ b/GALAHAD.jl/test/test_lhs.jl
@@ -19,7 +19,7 @@ function test_lhs(::Type{T}, ::Type{INT}) where {T,INT}
   # Parameters
   n_dimen = INT(7)  # dimension
   n_points = INT(2)  # points required
-  X = zeros(INT, n_points, n_dimen)  # points
+  X = zeros(INT, n_dimen, n_points)  # points
   seed = Ref{INT}()
 
   # Set a random seed
@@ -30,9 +30,9 @@ function test_lhs(::Type{T}, ::Type{INT}) where {T,INT}
 
   if inform[].status == 0 # successful return
     @printf("LHS successful\n")
-    for i in 1:n_points
-      @printf("Point %d = [", i)
-      for j in 1:n_dimen
+    for j in 1:n_points
+      @printf("Point %d = [", j)
+      for i in 1:n_dimen
         @printf("%d ", X[i,j])
       end
       @printf("]\n")

